lib/classes/PluginManager.js

```diff
@@ -352,6 +353,13 @@ class PluginManager {
     });
   }
 
+  assignDefaultOptions(command) {
+    _.forEach(command.options, (value, key) => {
+      if (value.default && (!this.cliOptions[key] || this.cliOptions[key] === true)) {
```

Comment:
`these_accusations === false && these_accusations !== true`

I have no idea how it got there :D I did a lot of cut pasting when I did that refactor.....